packages/components/src/Select.tsx

```diff
@@ -0,0 +1,263 @@
+import _ from 'lodash/fp';
+import { ReactNode } from 'react';
+import RSelect, {
+  components as RSelectComponents,
+  GroupBase as RSelectGroupBase,
+  MultiValue as RSelectMultiValue,
+  Props as RSelectProps,
+  PropsValue as RSelectPropsValue,
+  SingleValue as RSelectSingleValue,
+} from 'react-select';
+import  from 'react-select/async-creatable';
```

Comment:
It looks like the imported name got lost here.

```suggestion
import RAsyncCreatableSelect from 'react-select/async-creatable';
```